cmd/vic-machine/delete/delete.go

```diff
@@ -115,36 +115,39 @@ func (d *Uninstall) Run(cli *cli.Context) error {
 
 	validator, err := validate.NewValidator(ctx, d.Data)
 	if err != nil {
-		err = errors.Errorf("%s. Exiting...", err)
-		return err
+		log.Errorf("Delete cannot continue - failed to create validator: %s", err)
+		return errors.New("delete failed")
 	}
 	executor := management.NewDispatcher(validator.Context, validator.Session, nil, d.Force)
 
 	vch, _, err := executor.NewVCHFromComputePath(d.Data.ComputeResourcePath, d.Data.DisplayName, validator)
 	if err != nil {
 		log.Errorf("Failed to get Virtual Container Host %s", d.DisplayName)
-		return err
+		log.Error(err)
+		return errors.New("delete failed")
 	}
 	vchConfig, err := executor.GetVCHConfig(vch)
 	if err != nil {
-		log.Errorf("Failed to get Virtual Container Host configuration")
-		return err
+		log.Error("Failed to get Virtual Container Host configuration")
+		log.Error(err)
+		return errors.New("delete failed")
```

Comment:
On these errors, we should include all of the textual information in the error that's returned, because it gets concatenated together with other errors further down (up?) the stack for increasing contextual information. So instead of doing `log.Error(message` and then `return errors` you should collapse them into a single returned error I do believe